data/blog/electron-5-0.md

```diff
@@ -0,0 +1,113 @@
+
+
+Exposes an API to allow apps to determine their status as a trusted accessibility client. #16119
+Feat: provide user system's region with app.getLocaleCountryCode(). #15035
+
+Mixed-sandbox mode works on Linux. #15870
+The browser-backward and browser-forward app-command events events available in BrowserWindow now work on Linux. #15441
+Unified behavior between the default app and packaged apps (application menu / window-all-closed handling). #16310
+Updated SpellCheck API to support asynchronous results. #14032
+Upgraded to Chromium 71.0.3578.98. #15966
+Added --disable-color-correct-rendering switch. #15898
+
+Added win.removeMenu() to remove application menus instead of using win.setMenu(null). #16657
+Added caps lock and numlock as keyboard accelerator modifiers. #16725
+Converted debugger.sendCommand() to return a Promise instead of taking a callback. #16931
+Updated Menu.buildFromTemplate() to allow it to accept MenuItems in addition to plain objects. #16783
+Added webFrame.setIsolatedWorldInfo API and deprecated related APIs. #16932
+
+Enabled NodeIntegrationInSubFrames option usage for webview tags. #17398
+Enabled the setuid sandbox on Linux, allowing Electron to launch sandboxed processes in environments that disable CLONE_NEWUSER for unprivileged users. #17343
+
+
+## Breaking Changes
+
+### Breaking Change
+
+https://github.com/electron/electron/blob/master/docs/api/breaking-changes.md#planned-breaking-api-changes-50
+
+Mixed-sandbox mode is now enabled by default. enableMixedSandbox and the --enable-mixed-sandbox command-line switch still exist for compatibility, but are deprecated and have no effect. #15894
+The default values of nodeIntegration and webviewTag are now false to improve security. #16235
+
+## Deprecations
+
+The following breaking changes are planned for Electron 6.0, and thus are deprecated in Electron 5.0.
+
+### Deprecation
+
+https://github.com/electron/electron/blob/master/docs/api/breaking-changes.md#planned-breaking-api-changes-60
+
+Native versions of mksnapshot for arm and arm64 are deprecated.
+Deprecated ServiceWorker APIs on WebContents in preparation for their removal. #16732
+Deprecated modules internally using remote.require in sandboxed renderer context. #15145
+Added webFrame.setIsolatedWorldInfo API and deprecated related APIs. #16932
+enableMixedSandbox and the --enable-mixed-sandbox command-line switch still exist for compatibility, but are deprecated and have no effect. #15894
+
+## App Feedback Program
+
+The [App Feedback Program](https://electronjs.org/blog/app-feedback-program) we instituted during the development of Electron 3.0 was successful, so we've continued it during the development of 5.0 as well. To learn more about the App Feedback Program and to participate in future betas, [check out our blog post about the program](https://electronjs.org/blog/app-feedback-program).
+
+## What's Next
```

Comment:
I can get a 6.0.0 Timeline pushed up soon if we want to link to it in this section

Reply:
https://github.com/electron/electron/pull/17866/files